code/parse/sexp_container.cpp

```diff
@@ -37,6 +42,22 @@ const SCP_vector<list_modifier>& get_all_list_modifiers()
 	return List_modifiers;
 }
 
+// CTEXT()-related data
+namespace {
+	const char *Empty_str = "";
+} // namespace
+
+static bool str_prefix(const char* prefix, const char* str)
+{
+	for (; *prefix; ++prefix, ++str) {
+		if (SCP_tolower(*str) != SCP_tolower(*prefix)) {
+			return false;
+		}
+	}
+
+	return true;
+}
```

Comment:
Why not use `strnicmp`?

Reply:
Hmm, I guess I didn't know about `strnicmp()`. Will fix. Thanks.